docs/vgclog_balancedexamples.md

```diff
@@ -0,0 +1,643 @@
+
+|GC Operations | GC increment | *stop-the-world* or concurrent| XML element of GC increment          | Details                                                                   |
+|-------------|--------------|-------------------------------|--------------------------------------|---------------------------------------------------------------------------|
+|copy forward, optionally class unload, sweep, compact |Single        | *stop-the-world*              | `<gc-start>`, `<gc-end>`| `<gc-op'>` |ADD| 
+
+The `balanced` partial GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and some child elements are omitted for clarity:
+
+CHECK THIS ILLUSTRATION
+
+```xml
+<exclusive-start/>                       (STW pause starts)
+
+<allocation-taxation/>                   (memory threshold trigger recorded)
+
+<cycle-start/>                            (partial cycle starts)
+
+<gc-start/>                             (partial  cycle increment starts)
+
+<mem-info>                            (memory status before operations)
+
+ <mem></mem>                         (status of different types of memory)
+
+</mem-info>         
+
+</gc-start> 
+
+<allocation-stats>                      (snapshot of application threads status...
+                                          ... taken before the cycle starts)
+<gc-op> type="copy forward" </gc-op>    (copy forward operation completed)
+
+<gc-op> type="class unload" </gc-op>    (class unload operation completed)
+
+<gc-op> type="sweep" </gc-op>           (sweep operation completed)
+
+<gc-op> type="compact" </gc-op>         (compact operation completed)
+
+<gc-end>                                (partial cycle increment ends)
+
+<mem-info>                            (memory status after operations)
+          
+<mem></mem>                         (status of different types of memory)
+
+</mem-info>         
+
+</gc-end>
+
+<cycle-end>                               (partial cycle ends)
+
+<exclusive-end>                           (STW pause ends)
+```
+
+When the `balanced` partial GC cycle is triggered, the GC runs an STW pause. Application threads are halted to give the garbage collector exclusive access to the heap. The STW pause is recorded in the logs by the `<exclusive-start>` element. 
+
+```xml
+<exclusive-start id="184" timestamp="2021-02-26T11:11:42.310" intervalms="3745.790">
+  <response-info timems="3.138" idlems="1.056" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+```
+
+A `balanced` partial GC cycle is triggered when the region count for the *eden* space reaches a *taxation* threshold. At this threshold, the GC 'taxes' the application threads, who have been 'paid' with memory allocation, to run some GC work - in this case, a partial GC cycle.The logs record this trigger reason by using the`<allocation-taxation>` element.
```

Comment:
'who have been 'paid' with memory allocation, to run some GC work'

This actually applies more to taxation that occurs in Gencon's concurrent marking. There is taxation that occur behind the scene, that users do not need to know about.

For Balanced, taxes are not paid by running GC work in mutator threads, but just by virtue of stopping and letting dedicated GC threads do their work. Nothing really magical.